iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerLexicalGrammar.java

```diff
@@ -118,6 +118,13 @@ private static void punctuators(LexerlessGrammarBuilder b) {
 
   private static void lexical(LexerlessGrammarBuilder b) {
     b.rule(SPACING).is(
```

Comment:
I would suggest adding some tests to `DockerLexicalGrammarTest` to verify `SPACING` & `INSTRUCTION_SPACING`

Reply:
`SPACING` and `INSTRUCTION_SPACING` will be covered in `FileTreeImplTest` because skipped tokens can not be tested directly